src/main/java/com/epam/controller/MainController.java

```diff
@@ -1,17 +1,33 @@
 package com.epam.controller;
 
+import com.epam.DAO.PersonService;
+import com.epam.model.Person;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.ModelMap;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.servlet.ModelAndView;
 import java.security.Principal;
-
+import java.util.List;
 
 
 @Controller
 public class MainController {
+
+  private final PersonService personService;
+  @Autowired
+  public MainController(PersonService personService) {
+    this.personService = personService;
+  }
+
+
   @GetMapping("/index")
   public String defaultPage(Principal user, ModelMap model) {
+    List<Person> persons = personService.getPersons();
+    for(Person person: persons){
```

Comment:
If want to use java 8, can we use some new cool things, such as people.forEach() method. What do you think about it?

Reply:
https://habr.com/ru/post/337350/